src/main/steps/c100-rebuild/child-details/add-children/content.ts

```diff
@@ -0,0 +1,196 @@
+import { childrenDetails } from '../../../../app/controller/AppRequest';
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent, GenerateDynamicFormFields } from '../../../../app/form/Form';
+import { isFieldFilledIn } from '../../../../app/form/validation';
+import { C100_children_DETAILS_ADD } from '../../../urls';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  pageTitle: 'Enter the names of the children',
+  subTitle: 'Only include the children you’re making this application about',
+  firstName: 'First name(s)',
+  firstNameHint: 'Include all middle names here',
+  lastName: 'Last name(s)',
+  buttonAddChild: 'Add another child',
+  removeChild: 'Remove Child',
+  labelFornewName: 'Enter a new name',
+  errors: {
+    firstname: {
+      required: 'Enter the first name',
+    },
+    lastname: {
+      required: 'Enter the last name',
+    },
+  },
+});
+
+const cy = () => ({
+  pageTitle: 'Enter the names of the children- welsh',
+  subTitle: 'Only include the children you’re making this application about- welsh',
+  firstName: 'First name(s) - welsh',
+  firstNameHint: 'Include all middle names here - welsh',
+  lastName: 'Last name(s) - welsh',
+  buttonAddChild: 'Add another child - welsh',
+  removeChild: 'Remove child - welsh',
+  labelFornewName: 'Enter a new name - welsh',
+  errors: {
+    firstname: {
+      required: 'Enter the first name - welsh',
+    },
+    lastname: {
+      required: 'Enter the last name - welsh',
+    },
+  },
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const generateFormFields = (applicantData: childrenDetails[]): GenerateDynamicFormFields => {
```

Comment:
please rename it to listOfChildren as this is not applicantData

Reply:
it's been resolved